package.json

```diff
@@ -22,8 +22,8 @@
     "@polymer/app-layout": "^3.0.0",
     "@polymer/polymer": "^3.1.0",
     "@webcomponents/webcomponentsjs": "^2.2.4",
-    "lit-element": "^2.0.1",
-    "pwa-helpers": "^0.9.0",
```

Comment:
I prefer just refreshing package-lock.json when the app doesn't actually require features from the latest release (e.g. https://github.com/Polymer/pwa-starter-kit/commit/8dde11855060087b689186bc7b6c3b74446f62a2)

Reply:
@keanulee You are right! Fixed.